rivertype/river_type.go

```diff
@@ -214,9 +214,9 @@ type AttemptError struct {
 // subsequently remove the periodic job with `Remove()`.
 type PeriodicJobHandle int
 
-// Queue is a configuration for a queue that is currently (or recently was) in
+// QueueRow is a configuration for a queue that is currently (or recently was) in
 // use by a client.
-type Queue struct {
+type QueueRow struct {
```

Comment:
I'm hesitant to do this. We had to call the job one `JobRow` because we wanted the generic typed variant to be `Job[T]`. However we have no such issue with the queue type and I doubt we will, so it seems unnecessary to clutter this one up with a `*Row` suffix.

Reply:
Fair enough — maybe our convention can just be that every model gets its normal noun as a name, and `JobRow` is the single exceptional case. I reverted back to `Queue` and updated the commit message.